src/sentry/integrations/github_enterprise/payload.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import absolute_import
+
+import hashlib
+import hmac
+import logging
+import six
+
+from django.http import HttpResponse
+from django.utils.crypto import constant_time_compare
+from django.utils.decorators import method_decorator
+from django.views.decorators.csrf import csrf_exempt
+from simplejson import JSONDecodeError
+from sentry import options
+
+from sentry.utils import json
+from sentry.api.base import Endpoint
+
+logger = logging.getLogger('sentry.integrations.github-enterprise')
+
+
+class GitHubEnterpriseAppsEndpoint(Endpoint):
```

Comment:
Are these webhooks going to be so different that we have to use an entirely different endpoint?